api/services/locations.js

```diff
@@ -0,0 +1,170 @@
+import config from 'config';
+import AWS from 'aws-sdk';
+import { has, sortBy } from 'lodash';
+import { db } from './db';
+
+// Athena config
+const {
+  accessKeyId,
+  secretAccessKey,
+  OutputLocation,
+  Database,
+  region,
+  measurementsTable
+} = config.get('athena');
+
+// Athena client
+const athena = new AWS.Athena({
+  accessKeyId,
+  secretAccessKey,
+  region
+});
+
+// The locations query
+const QueryString = `
+  SELECT
```

Comment:
What's going to happen to locations with no coordinates? Are we accounting for them here somehow?